ontology/proco.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ontology_detail
+id: proco
+title: "PROCO: Ontology of Process Chemistry"
+contact:
+  email: wes_schafer@merck.com
+  label: Wes Schafer
+  github: schaferw
+  orcid: ?
+description: OPC covers process chemistry, the chemical field concerned with scaling up laboratory syntheses to commercially viable processes.
+domain: biochemistry
```

Comment:
@matentzn As PROCO stands now it is focused on pharmaceutical process development, for which biochemistry is appropriate.  Like CHMO and CHEMINF however, there is broad applicability to chemistry in general and I would suggest the more generic "Chemistry" domain be allowed in the future.  I suspect this would constitute submitting an issue.

Reply:
Yes it would need a separate ticket!